app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Recipes</h1>
+<% @recipes.each do |recipe| %>
+<a href="recipes/<%= recipe.id %>">Recipe #<%= recipe.id %></a>
+<div>Name: <%= recipe.name %> by <%= recipe.user.name %></div>
+<div>Description: <%= recipe.description %></div>
+<div>Public? <%= recipe.public %></div>
+<% RecipeFood.where(recipe: recipe).each do | recipe_food | %>
+<div>Total food items: <%= recipe_food.quantity %></div>
+<div>Total price: $<%= recipe_food.quantity * recipe_food.food.price %></div>
```

Comment:
- Please present the total of all foods in the recipe instead of data for each item. Following the wireframe is a project requirement.
- Kindly enter the total as zero when there is no food in the recipe. It creates a better UX and provides clearer information.
<img width="468" alt="Screen Shot 2022-10-05 at 9 16 19 AM" src="https://user-images.githubusercontent.com/47673311/194058240-a0539568-78ad-4364-a5bd-9cf6be469552.png">
<img width="591" alt="Screen Shot 2022-10-05 at 9 17 24 AM" src="https://user-images.githubusercontent.com/47673311/194059277-5b26dcd1-24bd-4305-bce5-36403d906c9c.png">